command/src/logging/logs.rs

```diff
@@ -120,12 +124,20 @@ impl Logger {
     pub fn init(
         tag: String,
         spec: &str,
-        backend: LoggerBackend,
+        log_target: &str,
         colored: bool,
-        access_backend: Option<LoggerBackend>,
+        access_logs_target: Option<&str>,
         access_format: Option<AccessLogFormat>,
         access_colored: Option<bool>,
     ) {
+        println!("setting log target to {log_target}");
+        let backend = target_or_default(log_target);
+
+        println!("setting target of access logs to {access_logs_target:?}");
+        let access_backend = access_logs_target.map(|target| {
+            target_to_backend(target).expect("could not setup logger for access logs")
```

Comment:
Could you provide a more precise error message?

Reply:
sure, will do

Reply:
wait, actually the `expect` statement does display the error furnished by `target_to_backend` so we have context.

Here is an example with a typo in the config value of `access_logs_target`:


```
could not setup logger for access logs: InvalidLogTarget("file:///vr/logs/sozu-access.log", "No such file or directory (os error 2)")
```